persistence/src/commonMain/db_user/com/wire/kalium/persistence/Users.sq

```diff
@@ -276,3 +276,7 @@ SELECT name, handle FROM User WHERE qualified_id = :userId;
 
 updateTeamId:
 UPDATE User SET team = ? WHERE qualified_id = ?;
+
+selectNameByMessageId:
+SELECT name FROM User
+WHERE qualified_id = (SELECT Message.sender_user_id FROM Message WHERE Message.id = :messageId AND Message.conversation_id = :conversationId LIMIT 1);
```

Comment:
```suggestion
WHERE qualified_id = (SELECT Message.sender_user_id FROM Message WHERE Message.id = :messageId AND Message.conversation_id = :conversationId);
```
The limit is redundant since the message + conversation id are PK